contributing/core_team.rst

```diff
@@ -205,6 +235,86 @@ following these rules:
 Getting the right category is important as it is used by automated tools to
 generate the CHANGELOG files when releasing new versions.
 
+.. tip::
+
+    Core team members are part of the ``mergers`` group on the ``symfony``
+    Github organization. This gives them write-access to many repositories,
+    including the main ``symfony/symfony`` mono-repository.
+
+    To avoid unintentional pushes to the main project (which in turn creates
+    new versions on Packagist), Core team members are encouraged to have
+    two clones of the project locally:
+
+    #. A clone for their own contributions, which they use to push to their
+       fork on GitHub. Clear out the push URL for the main repository using
+       ``git remote set-url --push origin dev://null``;
```

Comment:
What's the process around this? On my side, I do have an origin: my fork.

Reply:
Both are supported as far as I know. I've always learned to use `origin` for the original repository and use my username for my fork (also easy if you work with multiple forks).

Reply:
Maybe rework this then to:
"Ensure this clone doesn't have *any* remotes that point to the official symfony repo. This will prevent any unwanted push there."
or something similar?

Reply:
We still need a remote to be able to pull from the official repo. Only the push url must be removed.

Reply:
Ah true! On my side, my process is: use `git@githu.com` URL when I need push access, and `https://github.com` ones when readonly. I have credentials setup only for git access. Dunno if that applies to others, just sharing :)

Reply:
I've clarified it a bit :)